infra/core/env/scripts/Add-SshKeyToGitHubRepo.ps1

```diff
@@ -0,0 +1,213 @@
+﻿<#
+.SYNOPSIS
+     Adds the supplied GitHub Repository to an installed App Installation
+.DESCRIPTION
+    Adds the supplied GitHub Repository to an installed App Installation. Used for when App Installations are selected repos only.
+.PARAMETER AppIdSecretName
+    Mandatory. Github App Id Secret Name
+.PARAMETER AppKeySecretName
+    Mandatory. Github App Private Key Secret Name
+.PARAMETER Environment
+    Mandatory. Application Environment for Key Title
+.PARAMETER SSHPublicKeySecretName
+    Mandatory. Key Vault secret name for the SSH key
+.PARAMETER PSHelperDirectory
+    Mandatory. Directory Path of PSHelper module
+.EXAMPLE
+    .\Add-SshKeyToGitHubRepo.ps1 -AppId <AppId> -AppKey <AppKey> -Environment <Environment> -SSHPublicKeySecretName <SSHPublicKeySecretName> -PSHelperDirectory <PSHelperDirectory>
```

Comment:
I think the parameters names should be AppIdSecretName and AppKeySecretName.

Reply:
Fixed.